app/controllers/api/subcollections/tags.rb

```diff
@@ -84,11 +93,8 @@ def parse_tag(data)
       end
 
       def parse_tag_from_href(data)
-        href = data["href"]
-        tag  = if href && href.match(%r{^.*/tags/#{CID_OR_ID_MATCHER}$})
-                 klass = collection_class(:tags)
-                 klass.find(from_cid(href.split('/').last))
-               end
+        href = parse_href(data['href'])
```

Comment:
LGTM, although I can't tell from the context if we're saying here that this *needs* to be a tag url. @abellotti ?